Package.swift

```diff
@@ -3,7 +3,7 @@
 
 import PackageDescription
 
-let version = "4.7.0"
+let version = "4.10.0"
 let checksum = "50dd78d19f614a1cc727afdb90abf7ba978970993d616f058413e7b72d3b8bc1"
```

Comment:
Shouldn't the checksum be updated with the one of the newly built binaries which will be uploaded to the release?

Reply:
```
 - name: Update Package.swift
        run: |
          sed -i "s/let version = \".*\"/let version = \"${{ steps.get_version.outputs.VERSION }}\"/" Package.swift
          sed -i "s/let checksum = \".*\"/let checksum = \"$(shasum -a 256 opencv2.xcframework.zip | cut -d ' ' -f 1)\"/" Package.swift
```
@dkyurtov my understanding is the Github action will update it automatically (and also it looks like there is no need to update the version manually)

@imarinov3 could you please confirm if thats the case?

Reply:
Yes, it looks like this is the case.